src/pages/categories/Categories.jsx

```diff
@@ -2,11 +2,13 @@ import Container from '@mui/material/Container'
 import Box from '@mui/material/Box'
 
 import ExploreCategories from '~/containers/categories-page/explore-categories/ExploreCategories'
+import OfferRequestBlock from '~/containers/find-offer/OfferRequestBlock'
 
 const Categories = () => {
   return (
     <Box sx={ { backgroundColor: 'backgroundColor', flex: 1 } }>
-      <Container>
+      <Container sx={ { pt: 6 } } >
```

Comment:
do container as top component instead of box, and move sx to syles file

Reply:
@MickhaelSh  I think there are no need to move one styles to file

Reply:
for all make sense, if it is styles it should be in styles file, not everywhere

Reply:
fixed